src/pk/asn1/der/utf8/der_length_utf8_string.c

```diff
@@ -27,11 +27,17 @@ unsigned long der_utf8_charsize(const wchar_t c)
       return 1;
    } else if (c <= 0x7FF) {
       return 2;
+#if __WCHAR_MAX__ == 0xFFFF
+   } else {
+      return 3;
```

Comment:
I don't think that makes sense as a `wchar_t` on this platform can only hold 2 bytes... or did I miss something?